Assets/_BForBoss/_Core/Scripts/Boss/DerekContextManager.cs

```diff
@@ -125,9 +125,17 @@ private void HandleVulnerabilityExpiring(bool wasPhaseCompleted)
             _bossManager.UpdateVulnerability(_currentVulnerability);
         }
 
-        private void HandleDeath()
+        [Button]
+        public void HandleDeath()
         {
             Perigon.Utility.Logger.LogString("Player wins -> Start Defeat animation and then open thank you for playing Text box", LoggerColor.Green, "derekboss");
+            StateManager.Instance.SetState(State.EndGame);
+        }
+
+        [Button]
+        public void Test()
```

Comment:
If we want to keep this, should rename to "ForceDeath". If not, can just remove this